static/styles.css

```diff
@@ -30,6 +35,52 @@ main {
   width: 240px;
 }
 
+@media screen and (max-width: 500px) {
+  .Header > :not(.Header-item-nav){
+    display: none;
+  }  
+}
+
+.Footer {
+  /* background-color: var(--color-header-bg); */
```

Comment:
```suggestion
```
Commented code makes it hard to know whether it was commented for debugging purposes and should be re-enabled or if these are just leftovers from tests